sdk/search/search-documents/src/serviceUtils.ts

```diff
@@ -664,3 +665,18 @@ export function getRandomIntegerInclusive(min: number, max: number): number {
   const offset = Math.floor(Math.random() * (max - min + 1));
   return offset + min;
 }
+
+/**
+ * Helper method to create a SynonymMap object.
+ *
+ * @param name - Name of the SynonymMap.
+ * @param filePath - Path of the file that contains the Synonyms (seperated by new lines)
+ * @returns SynonymMap object
+ */
+export function createSynonymMapFromFile(name: string, filePath: string): SynonymMap {
+  const synonyms: string[] = fs.readFileSync(filePath, "utf-8").split("\n");
```

Comment:
I would much prefer we make this an async function and use `util.promisify` to make it work well with promises

Reply:
Done